src/encoded/schemas/user.json

```diff
@@ -65,6 +66,19 @@
         },
         "time_zone": {
             "$merge": "snovault:schemas/user.json#/properties/timezone"
+        },
+        "submits_for": {
+            "title": "Submits for",
+            "description": "Submission Centers a user is authorized to submit data for.",
+            "type": "array",
+            "permission": "restricted_fields",
+            "lookup": 70,
```

Comment:
What does this do? I eliminated many of these in encoded-core and when porting from elsewhere; perhaps that was a mistake.

Reply:
`lookup` is used to order fields in the submission view (when you edit an item on the portal), but this is a carry over from fourfront and could probably be removed, especially if all the others have. We don't use the submission view a lot but it can be useful in certain situations. Not sure we need to add them back at this time.